Sources/Client/Views/Play/MetalView.swift

```diff
@@ -28,8 +28,7 @@ struct MetalView {
     mtkView.framebufferOnly = true
     mtkView.clearColor = MTLClearColorMake(0.65, 0.8, 1, 1) // Sky colour
```

Comment:
Remove this clear color now because the off screen renderer handles that instead